.github/workflows/publish-docker-image-for-eth.yml

```
@@ -32,18 +32,18 @@ jobs:
           push: true
           tags:  blockscout/blockscout-${{ env.DOCKER_CHAIN_NAME }}:${{ env.RELEASE_VERSION }}-postrelease-${{ env.SHORT_SHA }}
           labels: ${{ steps.setup.outputs.docker-labels }}
-          platforms: |
+          platforms: | governance 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)